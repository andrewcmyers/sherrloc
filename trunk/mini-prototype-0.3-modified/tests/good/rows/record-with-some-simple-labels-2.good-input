let f x =  { l1 = x and l2 = ()} 

