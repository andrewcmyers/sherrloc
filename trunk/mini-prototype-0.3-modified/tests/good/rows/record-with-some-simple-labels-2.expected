val f: forall a. a -> { l1: pre a ; l2: pre unit ; \ abs }

